packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx

```diff
@@ -50,17 +51,24 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
   },
   usdcPurchase: {
     backgroundColor: palette.specialLightGreen
+  },
+  unlocked: {
+    borderRadius: spacing(3),
+    paddingVertical: spacing(0.5),
+    paddingHorizontal: spacing(2.5)
   }
 }))
 
 export const LineupTileAccessStatus = ({
   contentId,
   contentType,
-  streamConditions
+  streamConditions,
+  hasStreamAccess
 }: {
   contentId: ID
   contentType: PurchaseableContentType
   streamConditions: AccessConditions
+  hasStreamAccess: boolean | undefined
```

Comment:
`hasStreamAccess?` instead?

Reply:
I'm intentionally making it required, but the type coming in from above can be undefined, so this is where it landed